boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,39 +59,172 @@ currently supports the following hardware features:
    * - GPIO
      - on-chip
      - gpio
+   * - ADC
+     - on-chip
+     - adc
+   * - I2C
+     - on-chip
+     - i2c
+   * - KSCAN
+     - on-chip
+     - kscan
 
-Programming and debugging
-*************************
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the
+:zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
+
+Programming and debugging on it83202
+************************************
+
+In order to upload the application to the device,
+you'll need our flash tool and Download board.
+You can get them at:`ITE's website`_.
```

Comment:
Maybe add a space between `:` and the link? Seeing them joined together makes me feel annoyed, but maybe it's just me. This also happens in the previous reference to ITE website in the doc (which wasn't touched in this PR).

Reply:
Sorry, it's my fault. Fixed.